pkg/utils/k8sutil.go

```diff
@@ -282,6 +282,16 @@ func CreateK8sResources(ns, name string, spec *spec.FunctionSpec, client *kubern
 									MountPath: "/kubeless",
 								},
 							},
+							LivenessProbe: &v1.Probe{
+								InitialDelaySeconds: int32(3),
+								PeriodSeconds:       int32(1),
```

Comment:
Checking every second is very aggressive, and will quickly obscure any real traffic.  Back to default of 10s, or even 30s?